core/files/assets/files-upload-handler.php

```diff
@@ -31,7 +31,7 @@ private function is_elementor_media_upload() {
 	 * @return bool
 	 */
 	final public static function is_enabled() {
-		$enabled = ! ! get_option( self::OPTION_KEY ) && self::file_sanitizer_can_run();
+		$enabled = 'yes' === get_option( self::OPTION_KEY ) && self::file_sanitizer_can_run();
```

Comment:
It's not backward compatible.
Please remove all changes you make for the wp_options values.

Reply:
Handled.